activity: Update realm links on server admin activity page

laurynmm opened the pull request:
Updates the links on the activity page:
- Removes the copy link for a realm's owners. This is available on the support page.
- Adds a link to the realm's Zulip.
- Updates the realm's stats page link so that it is just the pie chart icon.
- Adds a link to the realm's support page.

---

**Screenshots and screen captures:**

*Note hovering so that the link url is shown in the bottom left of each screenshot*
<details>
<summary>Hover over the house icon link to the realm's Zulip</summary>

![Screenshot from 2022-12-01 17-07-38](https://user-images.githubusercontent.com/63245456/205102985-86817ae4-3b20-49a8-a2de-0e172a4b72fc.png)
</details>
<details>
<summary>Hover over the pie chart icon link to the realm's stats page</summary>

![Screenshot from 2022-12-01 17-07-46](https://user-images.githubusercontent.com/63245456/205102966-56aa009f-81ed-4bf4-b01b-b127c967b6f5.png)
</details>
<details>
<summary>Hover over the realm name to the realm's support page</summary>

![Screenshot from 2022-12-01 17-07-54](https://user-images.githubusercontent.com/63245456/205102937-cf324157-1dd7-4bdc-9e59-058ef3e03e96.png)
</details>

---

**Self-review checklist**

<!-- Prior to submitting a PR, follow our step-by-step guide to review your own code:
https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code -->

<!-- Once you create the PR, check off all the steps below that you have completed.
If any of these steps are not relevant or you have not completed, leave them unchecked.-->

- [x] [Self-reviewed](https://zulip.readthedocs.io/en/latest/contributing/code-reviewing.html#how-to-review-code) the changes for clarity and maintainability
      (variable names, code reuse, readability, etc.).

Communicate decisions, questions, and potential concerns.

- [ ] Explains differences from previous plans (e.g., issue description).
- [ ] Highlights technical choices and bugs encountered.
- [x] Calls out remaining decisions and concerns.
- [ ] Automated tests verify logic where appropriate.

Individual commits are ready for review (see [commit discipline](https://zulip.readthedocs.io/en/latest/contributing/version-control.html)).

- [x] Each commit is a coherent idea.
- [x] Commit message(s) explain reasoning and motivation for changes.

Completed manual review and testing of the following:

- [x] Visual appearance of the changes.
- [ ] Responsiveness and internationalization.
- [ ] Strings and tooltips.
- [ ] End-to-end functionality of buttons, interactions and flows.
- [ ] Corner cases, error conditions, and easily imagined bugs.

laurynmm commented on analytics/views/installation_activity.py:
```diff
@@ -187,19 +189,10 @@ def realm_summary_table(realm_minutes: Dict[str, float]) -> str:
-        role=UserProfile.ROLE_REALM_OWNER,
-        is_active=True,
-    ):
-        realm_owners[up.realm.string_id].append(up.delivery_email)
```
I went ahead and removed this whole query. These email addresses can also be copied / accessed on the realm's support page.

timabbott replied:
Sounds good.

timabbott commented on analytics/views/activity_common.py:
```diff
@@ -78,10 +78,23 @@ def realm_stats_link(realm_str: str) -> Markup:
 
+def realm_support_link(realm_str: str) -> Markup:
+
+    url = f"activity/support?q={realm_str}"
```
I think you want to use `append_url_query_string` for this.

laurynmm replied:
Yes, thanks! Updated to use that to add the query parameter to the url.

timabbott commented:
Posted a few comments; I'll let @alya review the UI.

The other thing we want to do with this page, BTW, is split the different tabs into independent pages. Right now, we do all the work to load and render all the tabs even though most of the time one just wants to look at one of them; this makes this slow and also just makes the code a lot harder to read. 

I'm not sure what the best design for doing that is, but one strategy would be to just change the "tabs" into links and give each its own URL. (Removing use of the bootstrap tabs component is probably net helpful).

alya commented:
> Posted a few comments; I'll let @alya review the UI.

I'm not sure what the links are supposed to be, so I don't know how to verify that, but the UI looks fine to me.

laurynmm commented:
**Updates**:
- Updated how the link to the support page for the realm is generated.
- Added a commit with these additional clean-ups I noticed today:
  - Fixed the string generating the UTC time on the installation summary page, so that the random 'Z' isn't there anymore.
  - Changed the header type in the installation summary page from `<h2>` to `<h3>` to match the other tab headers.
- There was an unused `realm_link` on the `activity.html` template. So, I added a final commit that updated the link so that on the realm activity pages, the title has the new pie chart icon link to the realm's stats page. If this isn't something we'd like to have, I can also just remove that unused parameter from the template.

**Screenshots**:

<details>
<summary>Installation activity page</summary>

*Note smaller header size and fixed timezone*
![Screenshot from 2022-12-02 14-47-51](https://user-images.githubusercontent.com/63245456/205309237-49084e40-9c5a-4bfe-8815-17ba65561bc9.png)
</details>
<details>
<summary>Realm activity page</summary>

*Note the new pie chart link*
![Screenshot from 2022-12-02 14-48-00](https://user-images.githubusercontent.com/63245456/205309215-cfd184b1-8c99-448d-8fa0-3b7a6e043321.png)
</details>
<details>
<summary>User activity page</summary>

![Screenshot from 2022-12-02 14-48-09](https://user-images.githubusercontent.com/63245456/205309178-3851ab2a-143c-453f-b7a8-8f2b0eb1b629.png)
</details>

---

> I'm not sure what the best design for doing that is, but one strategy would be to just change the "tabs" into links and give each its own URL. (Removing use of the bootstrap tabs component is probably net helpful).

Cool, I'll mark that down as a follow-up to look into.

alya commented:
Works for me!

timabbott commented:
Great, merged, thanks @laurynmm!
